docs/tutorials/index.rst

```diff
@@ -0,0 +1,11 @@
+Tutorials
+=======================================
+
+.. toctree::
+   :maxdepth: 1
+
+   dev-setup
+   blockchain     
+   first-transaction
+   proxy-tutorial
```

Comment:
Also, not part of the current changes, but this comment in the proxy tutorial is very important and should be put somewhere much more visible:
```
Off-chain transactions are stored in a sqlite database. You should do regular backups of this file – it contains balance signatures of the client, and if you lose them, you will have no way of proving that the client is settling the channel using less funds than he has actually paid to the proxy.
```